pkg/arvo/sys/vane/clay.hoon

```diff
@@ -2309,7 +2465,11 @@
     ^+  ..park
     ?^  err
       ((slog leaf+"clay: desk {<syd>} failed to unsuspend" u.err) ..park)
-    =.  liv.dom  %live
+    =.  liv.dom
+      ?:  ?=(agent-wefts:gall wef.dom)
+        %live
+      %-  (slog leaf+"clay: cannot revive {<syd>} at kelvin {<wef.dom>}" ~)
```

Comment:
the comment above this function says a failure to revive crashes the event, but this prints and no-ops -- I think we want it to !! instead